### PR TITLE
Validate and Sanitize Arguments 

### PR DESCRIPTION
**Pull request by vpatakottu**
Added logic to validate arguments. Many of the arguments, such as NamespaceBaseURI, already have validation/sanitization logic within GenerateSBOMAsync(). The only arg we need parsing so far is the Verbosity level.

### CODE REVIEW

File: src/Microsoft.Sbom.Targets/GenerateSbomTask.cs [hunk 1]
```diff
@@ -117,18 +119,12 @@ public GenerateSbomTask()
...
-            string.IsNullOrEmpty(PackageVersion))
-        {
-            Log.LogError("Required argument not provided.");
-            return false;
-        }
-
         try
         {
+            this.ValidateAndAssignVerbosity();
+            // Set other configurations. The GenerateSBOMAsyn() already sanitizes and checks for
```
Comment (gustavoaca1997) [inline src/Microsoft.Sbom.Targets/GenerateSbomTask.cs]:
```suggestion
            // Set other configurations. The GenerateSBOMAsync() already sanitizes and checks for
```

File: src/Microsoft.Sbom.Targets/GenerateSbomTask.cs [hunk 2]
```diff
@@ -163,4 +159,24 @@ public override bool Execute()
...
+
+    /// <summary>
+    /// Checks the user's input for Verbosity and assigns the
+    /// associated EventLevel value for logging.
+    /// </summary>
+    private EventLevel ValidateAndAssignVerbosity()
+    {
+        if (string.IsNullOrEmpty(this.Verbosity))
+        {
+            Log.LogMessage("No verbosity level specified. Setting verbosity level at \"LogAlways\"");
```
Comment (gustavoaca1997) [inline src/Microsoft.Sbom.Targets/GenerateSbomTask.cs]: You could use the expression `EventLevel.LogAlways` in the log message. Something like this:
```suggestion
            Log.LogMessage($"No verbosity level specified. Setting verbosity level at \"{EventLevel.LogAlways}\"");
```

File: src/Microsoft.Sbom.Targets/GenerateSbomTask.cs [hunk 3]
```diff
@@ -117,18 +119,12 @@ public GenerateSbomTask()
...
-            string.IsNullOrEmpty(PackageName) ||
-            string.IsNullOrEmpty(PackageVersion))
-        {
-            Log.LogError("Required argument not provided.");
-            return false;
-        }
-
         try
         {
+            this.ValidateAndAssignVerbosity();
```
Comment (gustavoaca1997) [inline src/Microsoft.Sbom.Targets/GenerateSbomTask.cs]: Is this call necessary?

**Review by gustavoaca1997 — CHANGES_REQUESTED** [review]

File: src/Microsoft.Sbom.Targets/GenerateSbomTask.cs (hunk 3 above)
Comment (vpatakottu) [reply src/Microsoft.Sbom.Targets/GenerateSbomTask.cs]: Oh oopsies, I forgot to delete that. Thanks for catching it!

File: src/Microsoft.Sbom.Targets/GenerateSbomTask.cs (hunk 1 above)
Comment (vpatakottu) [reply src/Microsoft.Sbom.Targets/GenerateSbomTask.cs]: Updated!

File: src/Microsoft.Sbom.Targets/GenerateSbomTask.cs (hunk 2 above)
Comment (vpatakottu) [reply src/Microsoft.Sbom.Targets/GenerateSbomTask.cs]: Updated!

File: src/Microsoft.Sbom.Targets/GenerateSbomTask.cs (hunk 3 above)
Comment (vpatakottu) [reply src/Microsoft.Sbom.Targets/GenerateSbomTask.cs]: Updated!

**Review by gustavoaca1997 — APPROVED** [review]

**Pull request merged**